_posts/2020-12-07-autumn-2020-newsletter.md

```diff
@@ -0,0 +1,103 @@
+---
+title: "Newsletter, autumn 2020"
+date: 2020-12-02
+author: Elena Findley-de Regt, Laura Scheske
+type: blogpost
+excerpt: Our focuses and flowstates for the autumn
+categories:
+  - News
+---
+
+
+# Foundation for Public Code newsletter - autumn 2020
+
+![illustration of hand with speech bubbles containing screws and bolts]({{site.url}}/assets/service-3.png)
+
+Welcome to our autumn newsletter, covering September - November 2020. We've had a focused, productive autumn, continuing our work with codebase communities and potential members.
+
+In the months ahead, we hope to announce new members and incubate a new codebase.
+
+You can [read our last newsletter](https://blog.publiccode.net/news/2020/12/02/2020-12-02-summer-2020-newsletter.html) (summer 2020) on our blog.
+
+## What we've been up to
+
+### Codebase stewardship
+
+The stewards team spent this autumn on community, tooling and governance support for the 2 codebases we're stewarding: Signalen and OpenZaak.
+
+#### Signalen
+
+We helped the Signalen community to collaborate better by setting up a mailing list for them in September. They're now using it to get a shared backlog online, so they're ready to start quarterly roadmaps in 2021.
+
+The [Signalen product website](https://signalen.org/en/) is now also available in English.
+
+In November, the Signalen community updated the [Standard for Public Code](https://standard.publiccode.net/) gap analysis included in their codebase community. [See the new version.](https://github.com/Amsterdam/signals/blob/master/docs/topics/signalen-and-standard-for-public-code.md)
+
+#### OpenZaak
+
+Throughout the autumn, the OpenZaak [mailing list](https://lists.publiccode.net/mailman/postorius/lists/openzaak-discuss.lists.publiccode.net/) has been [humming](https://lists.publiccode.net/hyperkitty/hyperkitty/list/openzaak-discuss@lists.publiccode.net/) - it's been a joy to see the community develop new norms and practices around working together.
+
+#### New work
+
+In October we released a [new version (0.2.0)](https://github.com/publiccodenet/standard/releases/tag/0.2.0) of the Standard for Public Code .
+
+Stewards held workshops with two development teams on how to apply the Standard in their codebase development and specifically in their environment.
+
+The codebase stewards are also exploring new opportunities:
+
+- meeting with 3 new codebase communities
```

Comment:
Should this line have a period or comma (or should the next bullet point get rid of it)?

Reply:
No terminal punctuation is needed because this is a bulleted list on the internet. Terminal punctuation slows down people skimming.